app/io/pathfinder/controllers/ClusterController.java

```diff
@@ -0,0 +1,104 @@
+package io.pathfinder.controllers;
```

Comment:
Reflection doesn't work so well on private methods. Which is what the routing table uses.